frontend/pages/_document.tsx

```diff
@@ -0,0 +1,32 @@
+import { IFIXIT_ORIGIN } from '@config/env';
+import Document, { Html, Head, Main, NextScript } from 'next/document';
+
+class MyDocument extends Document {
+   render() {
+      return (
+         <Html>
+            <Head>
+               <link
+                  href="https://fonts.googleapis.com/css2?family=Archivo+Black&family=Lato:wght@400;700&display=swap"
```

Comment:
Google dislikes any second paint above the fold. It's likely setting font-display: optional would fix it (https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display)
Optional will force the first load of "slow" users to not use our font(s) and thus fallback to the browser defaults.
It's possible to style the browser defaults close to the actual font, which is what we do on the main site. It's a pain to style and may not be worth it right now.

```suggestion
                  href="https://fonts.googleapis.com/css2?family=Archivo+Black&family=Lato:wght@400;700&display=optional"
```

Reply:
I think what we have here in this pull is an improvement, so let's kick this suggestion to a new issue